sdk/purview/purview-scanning-rest/test/public/dataSources.spec.ts

```diff
@@ -23,6 +23,7 @@ describe("List data sources", () => {
 
   it("should list all available data sources", async () => {
     const result = await client.path("/datasources").get();
+    console.log(result.status)
```

Comment:
Was this console.log left here on purpose?

Reply:
No I missed it, I just removed it 😄